docs/cli.md

```diff
@@ -62,11 +63,13 @@ which appears, for a catalog a user decides is titled nist-800-53, as:
 
 ```
 
-`dist` directory will contain the assembled version of the models located on the source model directories (at the project root level) which are: `catalogs`, `profiles`, `component-definitions`, `system-security-plans`, `assessment-plans`, `assessment-results` and `plan-of-action-and-milestones`. The assumption is that each of the OSCAL files within this folder is ready to be read by external 3rd party tools.
+`dist` directory will contain the assembled version of the top level models located on the source model directories which are: `catalogs`, `profiles`, `component-definitions`, `system-security-plans`, `assessment-plans`, `assessment-results` and `plan-of-action-and-milestones`. The assumption is that each of the OSCAL files within this folder is ready to be read by external 3rd party tools.
+
+In most of the places in the documentation we use `json` format for specifying model files, but they are equally applicable to `yaml` format also.
 
 ### Support for subdivided document structures
 
-The files constructed by OSCAL can run into the tens of thousands of lines of yaml or formatted json. At this size the
+The files constructed by OSCAL can run into tens of thousands of lines of yaml or formatted json. At this size the
 files become completely unmanageable for users. To combat this, trestle can `trestle split` a file into many smaller files and later merge those split files together.
```

Comment:
This may be a good place to mention that priority is to json and yaml is supported as best effort.

Reply:
specified above when talking about 2 formats.